src/attributes/type_system.md

```diff
@@ -64,30 +72,47 @@ Non-exhaustive types cannot be constructed outside of the defining crate:
 
 - Non-exhaustive variants ([`struct`][struct] or [`enum` variant][enum]) cannot be constructed
   with a [_StructExpression_] \(including with [functional update syntax]).
+- The visibility of the same-named constant of a [unit-like struct][struct]
+  is lowered to `min($vis, pub(crate))`.
+- The visibility of the same-named constructor function of a [tuple struct][struct]
+  is lowered to `min($vis, pub(crate))`.
```

Comment:
`min($vis, pub(crate))` doesn't really define what `min` or `$vis` means. I imagine most people can suss out what it means, but I would be more comfortable expressing it without inventing new syntax. Perhaps it could say something like:

> If the visibility of … is `pub`, its visibility is lowered to `pub(crate)`.

Reply:
I was copying the existing text from the matching section, but I agree it's not ideal. Edited both.